srcbook/README.md

```diff
@@ -11,17 +11,44 @@
   <a href="https://www.youtube.com/@srcbook">Youtube</a>
 </p>
 
+## Srcbook
+
+Srcbook is a TypeScript-centric app development platform. It allows you to create and iterate on web apps incredibly fast using AI as a pair-programmer.
+It can create or edit web apps, and also write and execute backend code through an interactive notebook interface.
+
+
+Srcbook is open-source (apache2) and runs locally on your machine. You need to bring your own API key for AI usage (we strongly recommend Anthropic with `claude-3-5-sonnet-latest`).
+
 ## Features
 
-- Create, run, and share reproducible programs and ideas
+### App Builder
+
+- AI app builder for TypeScript
+- Create, edit and run web apps
+- Use AI to generate the boilerplate, modify the code, and fix things
+- Edit the app with a hot-reloading web preview
+
+<picture>
+  <source media="(prefers-color-scheme: dark)" srcset="https://i.imgur.com/lLJPZOs.png">
+  <source media="(prefers-color-scheme: light)" srcset="https://i.imgur.com/k4xAyCQ.png">
+  <img alt="Example Srcbook" src="https://i.imgur.com/k4xAyCQ.png">
+</picture>
+
+### Notebooks
+
+- Create, run, and share TypeScript notebooks
 - Export to valid markdown format (.src.md)
 - AI features for exploring and iterating on ideas
 - Diagraming with [mermaid](https://mermaid.js.org) for rich annotations
 - Local execution with a web interface
 - Powered by Node.js
-- Open-source under the Apache2 license
 
-![Example Srcbook light](https://imagedelivery.net/oEu9i3VEvGGhcGGAYXSBLQ/ebfa2bfe-f805-4398-a348-0f48d4f93400/public)
```

Comment:
NPM doesn't have a darkmode so the lightmode is hardcoded. I don't think the <picture> element was supported on npm either (IIRC)

Reply:
Thanks, updated